src/supervisor-api.ts

```diff
@@ -114,10 +114,33 @@ export class SupervisorAPI {
 
 		this.api.use(authenticate(this.config));
 
-		this.api.post('/v1/blink', (_req, res) => {
+		this.api.post('/v1/blink', (req, res) => {
+			const blinks = checkInt(req.body.blinks) || undefined;
+			const onDuration = checkInt(req.body.onDuration) || undefined;
+			const offDuration = checkInt(req.body.offDuration) || undefined;
+			const pause = checkInt(req.body.pause) || undefined;
+			const timeout = checkInt(req.body.timeout) || undefined;
```

Comment:
Pretty sure you can just remove the `|| undefined`, as that's the return type for `checkInt`.